build/changelog/build-changelog.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+require __DIR__ . '/ChangelogBuilder.php';
+
+use Aws\Build\Changelog\ChangelogBuilder;
+
+$changelogBuilder = new ChangelogBuilder("", "", true);
```

Comment:
Ah, interestingly parameters inputs looks like this, it's a bit weird and not self-explanatory.

You could have a param array for the constructor, which would allow more flexible changes in the future.

Or, a easier short term way, change the order for constructer, to have `$verboseFlag` go first, then others could have default value as empty string. This will make constructor looks cleaner like: `new ChangelogBuilder(true);`